src/components/DropdownMenu.js

```diff
@@ -80,17 +82,17 @@ DropdownMenu.propTypes = {
   className: PropTypes.string,
 
   /**
-   * Label for the toggle
+   * Label for the toggle.
    */
   label: PropTypes.string,
 
   /**
-   * Direction in which to expand the dropdown.
+   * Direction in which to expand the dropdown. Default: 'down'.
    */
   direction: PropTypes.oneOf(['up', 'down', 'left', 'right']),
```

Comment:
Shall we change this to just 'down', 'left' or 'right' as discussed in #62 as part of this PR?